lsp-mode.el

```diff
@@ -69,9 +71,9 @@
             tcp-proc (open-network-stream (concat name " TCP connection")
                                           nil host port
                                           :type 'plain))
-      (set-process-query-on-exit-flag proc nil)
-      (set-process-query-on-exit-flag tcp-proc nil)
+      ;; TODO: Same :noquery issue (see above)
       (set-process-query-on-exit-flag (get-buffer-process (get-buffer stderr)) nil)
+      (set-process-query-on-exit-flag tcp-proc nil)
```

Comment:
This is unrelated to https://debbugs.gnu.org/cgi/bugreport.cgi?bug=30031, but I think `open-network-stream` should also have a `:noquery` keyword argument. Mind filing an Emacs bug?